src/test/java/edu/cornell/kfs/fp/document/CuDisbursementVoucherDocumentTest.java

```diff
@@ -593,24 +581,21 @@ public void setupMockObjects() {
         }
 
         private TransactionalDocumentPresentationController createMockPresentationController() {
-            TransactionalDocumentPresentationController presentationController = EasyMock.createMock(
-                    CuDisbursementVoucherDocumentPresentationController.class);
-            EasyMock.expect(presentationController.getEditModes(EasyMock.isA(CuDisbursementVoucherDocument.class)))
-                    .andStubReturn(presentationEditModes);
-            EasyMock.replay(presentationController);
-            return presentationController;
+            TransactionalDocumentPresentationController controller = Mockito.mock(CuDisbursementVoucherDocumentPresentationController.class);
+            Mockito.when(controller.getEditModes(Mockito.any())).thenReturn(presentationEditModes);
+            return controller;
         }
 
         private TransactionalDocumentAuthorizer createMockAuthorizer() {
-            TransactionalDocumentAuthorizer authorizer = EasyMock.createMock(TransactionalDocumentAuthorizer.class);
-            EasyMock.expect(authorizer.canEdit(EasyMock.isA(CuDisbursementVoucherDocument.class), EasyMock.eq(authorizedPerson)))
-                    .andStubReturn(Boolean.TRUE);
-            EasyMock.expect(authorizer.canEdit(EasyMock.isA(CuDisbursementVoucherDocument.class), EasyMock.not(EasyMock.eq(authorizedPerson))))
-                    .andStubReturn(Boolean.FALSE);
-            EasyMock.expect(authorizer.getEditModes(EasyMock.isA(CuDisbursementVoucherDocument.class), EasyMock.eq(authorizedPerson),
-                    EasyMock.eq(presentationEditModes))).andStubReturn(authorizationEditModes);
-            EasyMock.replay(authorizer);
-            return authorizer;
+            TransactionalDocumentAuthorizer documentAuthorizer = Mockito.mock(TransactionalDocumentAuthorizer.class);
+            Mockito.when(documentAuthorizer.canEdit(Mockito.any(), Mockito.any())).then(this::determineEditCall);
+            Mockito.when(documentAuthorizer.getEditModes(Mockito.any(), Mockito.any(),Mockito.any())).thenReturn(authorizationEditModes);
```

Comment:
The old mock of this method was checking for a much stricter set of arguments. Should the new code do so?